src/doc/unstable-book/src/language-features/cfg-version.md

```diff
@@ -0,0 +1,34 @@
+# `cfg_version`
+
+The tracking issue for this feature is: [#64796]
+
+[#64796]: https://github.com/rust-lang/rust/issues/64796
+
+------------------------
+
+The `cfg_version` feature makes it possible to execute different code
+depending on the compiler version.
+
+## Examples
+
+```rust
+#![feature(cfg_version)]
+
+#[cfg(version("1.42"))]
+fn a() {
+  // ...
```

Comment:
Should the examples here be 4 spaces?

Reply:
Oops, I just mostly copy-pasted from `cfg-sanitize.md` and didn't even notice. Thanks!